js/index.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+
+console.log('at doc ready');
+
+	$('.readless a').hide();
+	$('.readmore').click(articleOpen);
+	$('.readless').click(articleClose);
+	$('.learnmore').click(learnMoreOpen);
+
+
+	// article read more and read less
+	function articleOpen() {
+		$('#show-this-on-click').slideDown();
+		$('.readmore a').hide();
```

Comment:
This is absolutely correct, just keeping in mind by targeting the `a` element for hide/show you are leaving the parent element with the class(es) .readmore/.readless in your markup. You could also just target those directly so you don't have empty elements on your page.